examples/button.rb

```diff
@@ -1,7 +1,9 @@
+# frozen_string_literal: true
+
 Shoes.app do
   @push = button "Push me"
   @note = para "Nothing pushed so far"
-  @push.click {
+  @push.click do
```

Comment:
For idiomatic shoes I actually do like multiline curlies and I'm indifferent to frozen_string_literal: true. I think you did want to exclude the examples directory too?

Besides that, if examples are exempted and CI is green this is good to me :)